<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="5-Inventario_20pies_TestSuites" time="6.015" tests="2" failures="0" errors="1">
   <testsuite name="5-Inventario_20pies_TestSuites" tests="2" failures="0" errors="1" time="6.015" timestamp="2020-02-11 12:50:18" hostname="kaizen - LAPTOP-K43U1BVQ" id="Test Suites/Flujo_funcionalidades_Pedidos_20pies/5-Inventario_20pies_TestSuites">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="C:\\Users\\kaizen\\git\\AutomatizacionKatalon\\Reports\\20200211_125014\\Flujo_funcionalidades_Pedidos_20pies\\5-Inventario_20pies_TestSuites\\20200211_125014"/>
         <property name="logFiles" value="C:\\Users\\kaizen\\git\\AutomatizacionKatalon\\Reports\\20200211_125014\\Flujo_funcionalidades_Pedidos_20pies\\5-Inventario_20pies_TestSuites\\20200211_125014\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="kaizen - LAPTOP-K43U1BVQ"/>
         <property name="hostAddress" value="192.168.99.1"/>
         <property name="os" value="Windows 10 64bit"/>
         <property name="katalonVersion" value="7.2.1.1"/>
      </properties>
      <testcase name="Test Cases/Secuencias de Ejecucion/TC_Generar_Inventario_20pies" time="4.529" classname="Test Cases/Secuencias de Ejecucion/TC_Generar_Inventario_20pies" status="PASSED">
         <system-out><![CDATA[2020-02-11 12:50:18 - [TEST_CASE][PASSED] - Test Cases/Secuencias de Ejecucion/TC_Generar_Inventario_20pies: Test Cases/Secuencias de Ejecucion/TC_Generar_Inventario_20pies

2020-02-11 12:50:19 - [TEST_STEP][PASSED] - python.Ex_inventario_pedidos_20pies.runPython(): python.Ex_inventario_pedidos_20pies.runPython is PASSED

2020-02-11 12:50:23 - [MESSAGE][PASSED] - python.Ex_inventario_pedidos_20pies.runPython is PASSED]]></system-out>
         <system-err><![CDATA[]]></system-err>
      </testcase>
      <testcase name="Test Cases/Individual_Tests_20pies/Inventario" time="0.74" classname="Test Cases/Individual_Tests_20pies/Inventario" status="ERROR">
         <error type="ERROR" message="Test Cases/Individual_Tests_20pies/Inventario FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx
	at obtener_excels.obtener_inventario_20pies.invokeMethod(obtener_inventario_20pies.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at Inventario.run(Inventario:19)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1581439814437.run(TempTestSuite1581439814437.groovy:36)
Caused by: java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx
	at com.kms.katalon.core.testdata.reader.ExcelFactory.getExcelData(ExcelFactory.java:25)
	at com.kms.katalon.core.testdata.reader.ExcelFactory.getExcelDataWithDefaultSheet(ExcelFactory.java:17)
	at com.kms.katalon.core.testdata.reader.ExcelFactory$getExcelDataWithDefaultSheet.call(Unknown Source)
	at obtener_excels.obtener_inventario_20pies.getValuesExcelInventary(obtener_inventario_20pies.groovy:32)
	... 16 more
"/>
         <system-out><![CDATA[2020-02-11 12:50:23 - [TEST_CASE][ERROR] - Test Cases/Individual_Tests_20pies/Inventario: Test Cases/Individual_Tests_20pies/Inventario FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx
	at obtener_excels.obtener_inventario_20pies.invokeMethod(obtener_inventario_20pies.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at Inventario.run(Inventario:19)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1581439814437.run(TempTestSuite1581439814437.groovy:36)
Caused by: java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx
	at com.kms.katalon.core.testdata.reader.ExcelFactory.getExcelData(ExcelFactory.java:25)
	at com.kms.katalon.core.testdata.reader.ExcelFactory.getExcelDataWithDefaultSheet(ExcelFactory.java:17)
	at com.kms.katalon.core.testdata.reader.ExcelFactory$getExcelDataWithDefaultSheet.call(Unknown Source)
	at obtener_excels.obtener_inventario_20pies.getValuesExcelInventary(obtener_inventario_20pies.groovy:32)
	... 16 more


2020-02-11 12:50:24 - [TEST_STEP][ERROR] - data = obtener_excels.obtener_inventario_20pies.getValuesExcelInventary(Index): java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx

2020-02-11 12:50:24 - [MESSAGE][ERROR] - java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx]]></system-out>
         <system-err><![CDATA[2020-02-11 12:50:23 - [TEST_CASE][ERROR] - Test Cases/Individual_Tests_20pies/Inventario: Test Cases/Individual_Tests_20pies/Inventario FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx
	at obtener_excels.obtener_inventario_20pies.invokeMethod(obtener_inventario_20pies.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at Inventario.run(Inventario:19)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1581439814437.run(TempTestSuite1581439814437.groovy:36)
Caused by: java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx
	at com.kms.katalon.core.testdata.reader.ExcelFactory.getExcelData(ExcelFactory.java:25)
	at com.kms.katalon.core.testdata.reader.ExcelFactory.getExcelDataWithDefaultSheet(ExcelFactory.java:17)
	at com.kms.katalon.core.testdata.reader.ExcelFactory$getExcelDataWithDefaultSheet.call(Unknown Source)
	at obtener_excels.obtener_inventario_20pies.getValuesExcelInventary(obtener_inventario_20pies.groovy:32)
	... 16 more]]></system-err>
      </testcase>
      <system-out><![CDATA[2020-02-11 12:50:18 - [TEST_SUITE][ERROR] - 5-Inventario_20pies_TestSuites: Test Cases/Individual_Tests_20pies/Inventario FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx
	at obtener_excels.obtener_inventario_20pies.invokeMethod(obtener_inventario_20pies.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at Inventario.run(Inventario:19)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1581439814437.run(TempTestSuite1581439814437.groovy:36)
Caused by: java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx
	at com.kms.katalon.core.testdata.reader.ExcelFactory.getExcelData(ExcelFactory.java:25)
	at com.kms.katalon.core.testdata.reader.ExcelFactory.getExcelDataWithDefaultSheet(ExcelFactory.java:17)
	at com.kms.katalon.core.testdata.reader.ExcelFactory$getExcelDataWithDefaultSheet.call(Unknown Source)
	at obtener_excels.obtener_inventario_20pies.getValuesExcelInventary(obtener_inventario_20pies.groovy:32)
	... 16 more]]></system-out>
      <system-err><![CDATA[2020-02-11 12:50:18 - [TEST_SUITE][ERROR] - 5-Inventario_20pies_TestSuites: Test Cases/Individual_Tests_20pies/Inventario FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx
	at obtener_excels.obtener_inventario_20pies.invokeMethod(obtener_inventario_20pies.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at Inventario.run(Inventario:19)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1581439814437.run(TempTestSuite1581439814437.groovy:36)
Caused by: java.io.FileNotFoundException: C:/Users/kaizen/git/AutomatizacionKatalon/Pedidos/DataExcel/Inventario.xlsx
	at com.kms.katalon.core.testdata.reader.ExcelFactory.getExcelData(ExcelFactory.java:25)
	at com.kms.katalon.core.testdata.reader.ExcelFactory.getExcelDataWithDefaultSheet(ExcelFactory.java:17)
	at com.kms.katalon.core.testdata.reader.ExcelFactory$getExcelDataWithDefaultSheet.call(Unknown Source)
	at obtener_excels.obtener_inventario_20pies.getValuesExcelInventary(obtener_inventario_20pies.groovy:32)
	... 16 more]]></system-err>
   </testsuite>
</testsuites>
